litho-processor/src/main/java/com/facebook/litho/specmodels/model/EventValidation.java

```diff
@@ -60,6 +62,13 @@
                 "Event declarations must be annotated with @Event."));
       }
 
+      final boolean isKotlinClass = SpecElementTypeDeterminator.determine(
+          (TypeElement) eventDeclaration.representedObject) == SpecElementType.KOTLIN_CLASS;
+
+      if (isKotlinClass) {
+        continue;
```

Comment:
I think we also need to skip this if doing the Java ctor method right? Ideally we'd actually validate that it has the appropriate getters for the fields in the ctor. Even for the kotlin one we might want to validate there's only one ctor since we use the first one in `hasNonEmptyConstructors`